qlib/model/trainer.py

```diff
@@ -329,12 +339,24 @@ def train(
         run_task(
             train_func,
             task_pool,
+            query={"filter": {"$in": tasks}},  # only train these tasks
             experiment_name=experiment_name,
             before_status=before_status,
             after_status=after_status,
             **kwargs,
         )
 
+        # FIXME: reset to waiting automatically
+        for _id in _id_list:
+            is_prn = False
```

Comment:
I think the following logic will be simpler.
```suggestion
            if   tm.re_query(_id)["status"] == TaskManasger.RUNNING:
                  give the warning.
```